bower.json

```diff
@@ -22,36 +22,27 @@
     "/tests/"
   ],
   "dependencies": {
-    "app-route": "PolymerElements/app-route#^0.9.3",
-    "iron-ajax": "PolymerElements/iron-ajax#^1.0.0",
-    "marked-element": "PolymerElements/marked-element#^1.0.0",
-    "paper-button": "PolymerElements/paper-button#^1.0.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.5.0",
-    "prism-element": "PolymerElements/prism-element#^1.0.2"
+    "polymer": "Polymer/polymer#1.9 - 2",
+    "app-route": "PolymerElements/app-route#1 - 2",
+    "marked-element": "PolymerElements/marked-element#1 - 2",
+    "paper-button": "PolymerElements/paper-button#1 - 2",
+    "paper-styles": "PolymerElements/paper-styles#1 - 2",
+    "prism-element": "PolymerElements/prism-element#1 - 2"
   },
   "devDependencies": {
-    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
-    "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+    "iron-ajax": "PolymerElements/iron-ajax#1 - 2",
+    "iron-component-page": "PolymerElements/iron-component-page#1 - 2",
+    "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
+    "web-component-tester": "Polymer/web-component-tester#^6.0.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0"
   },
   "variants": {
-    "2.x": {
+    "1.x": {
       "dependencies": {
-        "app-route": "PolymerElements/app-route#2.0-preview",
-        "iron-ajax": "PolymerElements/iron-ajax#2.0-preview",
```

Comment:
The tests pass -- the 1.x variant runs with polymer 1.9.1 and e.g. paper-button 2.0.0. Isn't that right? The 2.0 polymer elements should be the same under both variants since they are hybrid, the only difference should be the polymer version?